### PR TITLE
Multistage build

### PR DESCRIPTION
**Pull request by rudigiesler**
Adds a build stage to the docker image, so that we don't need all the build requirements in the run image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -52,12 +34,37 @@ RUN /venv/bin/pip install --upgrade pip && poetry install --no-interaction --no-
...
         postgresql-client \
         libmagic-dev \
-        libpcre3
+        libpcre3 \
+        libgeos-c1v5 \
+        libgdal28 \
+        libproj19
+
+RUN apt-get-install.sh curl && \
+    curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
```
Comment (jerith) [inline Dockerfile:61]: Can we try using the version of node in the os repos? 10.x has been unsupported for nearly a year and 12.x only has a couple of months left.

Comment (rudigiesler) [reply Dockerfile]: I'm going to try and see if there are any breakages with nodejs 16 (the LTS version)

Comment (rudigiesler) [reply Dockerfile]: Seems like it works with nodejs 16

**Review by jerith — APPROVED** [review]
:+1:

**Pull request merged**